migrations/1685458879107.do.add-retrieval-result-columns.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE retrieval_results
+  ADD COLUMN start_at TIMESTAMP,
+  ADD COLUMN status_code INTEGER,
+  ADD COLUMN first_byte_at TIMESTAMP,
+  ADD COLUMN end_at TIMESTAMP,
```

Comment:
I think we should use `timestampz` to include the timezone information in the stored value.

See https://www.postgresql.org/docs/current/datatype-datetime.html